tests/e2e/defaultvmgroup_test.go

```diff
@@ -57,38 +62,114 @@ var _ = Suite(&DefaultVMGroupTestSuite{})
+	c.Assert(isVmgroupAvailable, Equals, true, Commentf(" Vmgroup %s does not exist", admincliconst.DefaultVMgroup))
+
+	// Create a volume
+	out, err := dockercli.CreateVolume(s.config.DockerHosts[0], s.volumeNames[0])
+	c.Assert(err, IsNil, Commentf(out))
+
+	// Check if volume was successfully created
+	isAvailable := verification.CheckVolumeAvailability(s.config.DockerHosts[0], s.volumeNames[0])
+	c.Assert(isAvailable, Equals, true, Commentf("Volume %s is not available after creation", s.volumeNames[0]))
+
+	// Verify if volume exists in default vmgroup
+	isVolInVmgroup := admincli.IsVolumeExistInVmgroup(s.config.EsxHost, s.volumeNames[0], admincliconst.DefaultVMgroup)
+	c.Assert(isVolInVmgroup, Equals, true, Commentf("Volume [%s] does not belong to vmgroup [%s]", s.volumeNames[0], admincliconst.DefaultVMgroup))
+
+	// Delete default vmgroup and verify it does not exist
+	admincli.DeleteVMgroup(s.config.EsxHost, admincliconst.DefaultVMgroup)
+
+	isVmgroupAvailable = admincli.IsVmgroupPresent(s.config.EsxHost, admincliconst.DefaultVMgroup)
+	c.Assert(isVmgroupAvailable, Equals, false, Commentf("Failed to delete vmgroup %s .", admincliconst.DefaultVMgroup))
+
+	// Create a volume - operation should fail as default vmgroup no longer exist
+	out, err = dockercli.CreateVolume(s.config.DockerHosts[0], s.volumeNames[1])
+	c.Assert(err, Not(IsNil), Commentf(out))
+	c.Assert(strings.Contains(out, noVgErrorMsg), Equals, true, Commentf(out))
+
+	// Verify existing volume - volumeName1 does not belong to any vmgroup (N/A)
+	isVolInVmgroup = admincli.IsVolumeExistInVmgroup(s.config.EsxHost, s.volumeNames[0], admincliconst.NotApplicable)
+	c.Assert(isVolInVmgroup, Equals, true, Commentf("Unexpected Behavior: Vmgroup for volume [%s] is not N/A. ", s.volumeNames[0]))
+
+	isVmgroupAvailable = admincli.CreateDefaultVmgroup(s.config.EsxHost)
+	c.Assert(isVmgroupAvailable, Equals, true, Commentf("Failed to create vmgroup %s .", admincliconst.DefaultVMgroup))
+
+	// Create a volume - operation should succeed as default vmgroup exists now
+	out, err = dockercli.CreateVolume(s.config.DockerHosts[0], s.volumeNames[1])
+	c.Assert(err, IsNil, Commentf(out))
+
+	// Check if volume was successfully created
+	isAvailable = verification.CheckVolumeAvailability(s.config.DockerHosts[0], s.volumeNames[1])
+	c.Assert(isAvailable, Equals, true, Commentf("Volume %s is not available after creation", s.volumeNames[1]))
+
+	// Check volumes now again belong to default vmgroup
+	c.Assert(s.isVolumeInVmgroup(s.config.EsxHost, s.volumeNames), Equals, true, Commentf("Volume [%s] does not belong to vmgroup [%s]", s.volumeNames, admincliconst.DefaultVMgroup))
+
+	out, err = ssh.InvokeCommand(s.config.DockerHosts[0], dockerconst.RemoveVolume+strings.Join(s.volumeNames, " "))
+	c.Assert(err, IsNil, Commentf(out))
+
+	misc.LogTestEnd(c.TestName())
+}
+
+func (s *DefaultVMGroupTestSuite) isVolumeInVmgroup(esxIP string, volumeNames []string) bool {
```

Comment:
We don't really need a receiver here. I'd suggest to remove the receiver so that we can move this private function to common util whenever needed.